packages/SwingSet/src/kernel/kernel.js

```diff
@@ -959,6 +960,10 @@ export default function buildKernel(
       return results;
```

Comment:
Do we also want something before this early return when `startVat` fails?
```suggestion
      console.log(
        `vat ${vatID} failed to upgrade from incarnation ${oldIncarnation} to ${newIncarnation} with source ${bundleID}: ${errorCapData}`,
      );
      return results;
```

Reply:
Hm, not a bad idea. Although `${errorCapData}` is likely to render as `[object Object]`, unless the logger we're using here is actually something more clever. I'll try it, but I may land it without the error data, and expect that folks will look in the slog for the details.

Reply:
indeed I got `[object Object]`, so I'll omit that